composer.json

```diff
@@ -1,12 +1,37 @@
 {
-    "repositories": [
-        {
-            "type": "vcs",
-            "url": "https://github.com/momentohq/client-types-php"
-        }
-    ],
+    "name": "momento/client-sdk-php",
+    "type": "library",
+    "autoload": {
+        "psr-4": {
+            "Momento\\": "src/",
+            "Cache_client\\": "types/Cache_client/",
+            "Control_client\\": "types/Control_client/",
+            "Auth\\": "types/Auth/",
+            "GPBMetadata\\": "types/GPBMetadata"
+        },
+        "files": [
+            "src/Utilities/_DataValidation.php"
+        ],
+        "classmap": [
+            "src/Cache/CacheOperationTypes/CacheOperationTypes.php"
+        ]
+    },
+    "autoload-dev": {
+        "psr-4": { "Momento\\Tests\\": "tests/" }
+    },
     "require": {
-        "momentohq/client-types-php": "dev-main"
+        "php": ">=7.4",
+        "google/protobuf": "^3.21",
+        "grpc/grpc": "^1.42",
+        "ext-grpc": "*"
```

Comment:
is it possible to keep all libraries pinned explicitly?

Reply:
Probably, but they're pinned in the composer.lock file already, so those versions will be what customers get until we regenerate it (now that I've corrected the README). I'd rather let these float in composer.json so we can receive updates without having to constantly monitor for them. I'm happy to discuss if you have any objection to that.

Reply:
Ok, I generally prefer to keep pinned here and then have something like dependabot open PR's back in bumping versions. Just helps keep library upgrades self-contained and helps us merge them on our terms and never accidentally as part of another unrelated pr. We're a little inconsistent on this across repos though right now so lets maybe sync offline can come back to it.

Reply:
I'm okay with coming back to this later, doesn't have to be part of this PR, but I agree with Ellery - we should pin explicitly.

Reply:
Dependabot is currently only checking versions on Momento libraries so wouldn't catch other library versions getting bumped as currently set up. I went ahead and pinned these in `composer.json` but we'll probably need to discuss this later.

Reply:
yep happy to discuss.